trade-finance-manager-api/package.json

```diff
@@ -44,6 +44,7 @@
     "crypto": "^1.0.1",
     "date-fns": "^2.30.0",
     "dotenv": "16.0.3",
+    "escape-string-regexp": "^4.0.0",
```

Comment:
Do we need to use this package, it's very small and and increases deps/security concerns etc, can we have our own implementation?

Reply:
I'd agree -- oddly this package has caused a fair bit of pain. We have it elsewhere however. I'll raise a ticket to look at removing this -- We have to be considerate to write our own implimentation of escaping strings, as this is published with a MIT licience, so we cannot simply copy and paste :)